tests/BringgTrackingClientTests.m

```diff
@@ -0,0 +1,550 @@
+//
+//  BringgTrackingTests.m
```

Comment:
for the whole class its about 45%
for the watch methods its almost 100% (start watch waypoint has no tests)
if you wish to increase the coverage for methods that are outside the scope of this PR - create an asana for it